src/EFCore.PG/Storage/Internal/Mapping/NpgsqlGeometricTypeMapping.cs

```diff
@@ -35,7 +36,7 @@ protected override RelationalTypeMapping Clone(RelationalTypeMappingParameters p
         protected override string GenerateNonNullSqlLiteral(object value)
         {
             var line = (NpgsqlLine)value;
-            return $"LINE '{{{line.A.ToString("G17", CultureInfo.InvariantCulture)},{line.B.ToString("G17", CultureInfo.InvariantCulture)},{line.C.ToString("G17", CultureInfo.InvariantCulture)}}}'";
+            return FormattableString.Invariant($"LINE '{{{line.A:G17},{line.B:G17},{line.C:G17}}}'");
```

Comment:
Something here isn't being interpolated per AppVeyor.

Reply:
Made it a little bit more readable.